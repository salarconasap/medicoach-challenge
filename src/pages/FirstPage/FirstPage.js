import { CustomLink } from '../../styles'

function FirstPage() {
  return (
    <CustomLink to="secondpage">
      Scan QR Code
    </CustomLink>
  );
}

export default FirstPage;
