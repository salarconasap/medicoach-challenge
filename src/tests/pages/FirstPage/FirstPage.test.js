import React from 'react'
import '@testing-library/jest-dom'
import { shallow } from 'enzyme';
import FirstPage from '../../../pages/FirstPage/FirstPage';

describe('test <FirstPage /> ', () => {

    const wrapper = shallow(<FirstPage />)

    test('should show FirstPage correctly', () => {
         expect(wrapper).toMatchSnapshot();
    })

})

