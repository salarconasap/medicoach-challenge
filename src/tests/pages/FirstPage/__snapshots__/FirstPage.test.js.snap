// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <FirstPage />  should show FirstPage correctly 1`] = `
<Styled(Link)
  to="secondpage"
>
  Scan QR Code
</Styled(Link)>
`;
